scripts/release/rpm/test_mariner_in_docker.sh

```diff
@@ -14,7 +14,7 @@ time az self-test
 time az --version
 
 cd /azure-cli/
-pip install wheel
+python -m pip install --upgrade pip setuptools
```

Comment:
Previously, `python3-setuptools` are installed by `python3-devel` package. The `pip` and `setuptools` are not latest version.

In Azure Linux 3, `setuptools` is not installed automatically. Use `pip` to explicitly install latest `pip` and `setuptools`

`wheel` is not a required package.